docs/embedding/static-embedding-parameters.md

```diff
@@ -165,6 +165,17 @@ You can preview appearance settings from your question or dashboard's [embedded
 
 \* Available on [Pro](https://www.metabase.com/product/pro) and [Enterprise](https://www.metabase.com/product/enterprise) plans.
 
+### Download results
+
+{% include plans-blockquote.html feature="Downloading results" %}
+
+You can enable or disable downloading results for embedded questions by setting the `hide_download_button` parameter in the embedding URL to `false` or `true`.
```

Comment:
We should say that the default is to show the download button. 

```suggestion
By default, Metabase will include a **Download** button on embedded questions. You can remove the download button by setting `hide_download_button=true` in the embedding URL in the iframe's `src` attribute.
```
And can you link to the [appearance section](https://www.metabase.com/docs/latest/embedding/static-embedding#customizing-the-appearance-of-static-embeds)? 

![image](https://github.com/metabase/metabase/assets/17343328/fce7b1b9-d533-4afe-8821-48df0d81dc2f)